variables.tf

```diff
@@ -55,6 +43,57 @@ variable "access_tags" {
   }
 }
 
+##############################################################################
+# Naming Variables
+##############################################################################
+
+variable "prefix" {
+  description = "The value that you would like to prefix to the name of the resources provisioned by this module. Explicitly set to null if you do not wish to use a prefix."
```

Comment:
Also add this:
This value is ignored is using one of the optional variables for explicit control over naming